amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -43,7 +43,11 @@ describe('SearchPage', () => {
       searchAll: jest.fn(),
       searchResource: jest.fn(),
       getPopularTables: jest.fn(),
-      ...propOverrides
+      history: mock,
```

Comment:
It's a better practice to declare these each individually as `jest.fn()` instead of reusing the same object. The reason is that in the event that we need to mock a specific functionality of any of these objects, we'd end up applying that change to all of these parameters instead of just the one it was intended for.